lcls-twincat-pmps/PMPS/AncillaryComponents/FB_JsonDocToSafeBP.TcPOU

```diff
@@ -79,7 +79,7 @@ CASE Step OF
 		
 	 21: (*Check device States in DOM*)
 	 		FOR Index:=1 TO MAX_STATES BY 1 DO
-				  IF NOT ((arrStates[Index].sName= '') OR (arrStates[Index].sName= 'Invalid')) THEN
+				  IF NOT ((arrStates[Index].sPmpsState= '') OR (arrStates[Index].sPmpsState= 'Invalid')) THEN
```

Comment:
These need to be `sName` or else the comparison doesn't work properly. `sPmpsState` will never be `Invalid`.